docs/Developer-guide/compiling-rpms.md

```diff
@@ -6,17 +6,18 @@ Creating rpm's of GlusterFS from git source is fairly easy, once you know the st
 RPMS can be compiled on at least the following OS's:
 
 - Red Hat Enterprise Linux 5, 6 (& 7 when available)
-- CentOS 5, 6 (& 7 when available)
+- CentOS 5, 6, and 8
```

Comment:
Why are  we removing CentOS 7 from the list? Is it no longer supported?

Reply:
No..no.. I must have removed it by mistake while editing it. My bad.